components/autocomplete/src/autocomplete-trigger.ts

```diff
@@ -220,7 +220,7 @@ export class DtAutocompleteTrigger<T>
         switchMap(() => this.optionSelections),
       );
     },
-  );
+  ) as Observable<DtOptionSelectionChange<T>>;
```

Comment:
typed because `defer()` function return `Obserable<unkown>`

Reply:
![image](https://user-images.githubusercontent.com/2905693/72133251-3ae36400-3381-11ea-8fe5-d456b6259dc4.png)
Works like a charm without the as operator. Can we please try to remove it and see if it breaks :) ?